libs/dh/admin/feature-user-management/src/lib/overview/dh-users-overview.component.html

```diff
@@ -35,21 +35,12 @@ <h3>{{ t("tabLabel") }}</h3>
     </vater-stack>
 
     <vater-stack direction="row" gap="m">
-      <dh-users-overview-status-filter
-        [initialValue]="initialStatusFilter$ | push"
-        (changed)="onStatusChanged($event)"
-      />
-
-      @if (canChooseMultipleActors$ | push) {
-        <dh-users-overview-actor-filter
-          [actorOptions]="actorOptions$ | push"
-          (changed)="onActorFilterChanged($event)"
-        />
-      }
-
-      <dh-users-overview-userrole-filter
+      <dh-users-overview-filters
+        [statusValue]="initialStatusFilter$ | push"
```

Comment:
Nitpick: isnt initialValue, or initialStatusValue better?

Reply:
Or maybe it is just filters now?

Reply:
`initialStatusValue` sounds right :)